src/test/java/com/google/showcase/v1beta1/BUILD.bazel

```diff
@@ -0,0 +1,48 @@
+load(
```

Comment:
Can we move all showcase related code to a separate folder like `test/showcase`? We plan to have no bazel stuff in the `src` folder, not sure if we can achieve the same thing with Maven. I also think it's fine to leave all the golden integration tests and showcase integration tests with Bazel, because the main benefit of migrating the generator to Maven is to have a hermetic code generation process, there is not much benefit of migration the testing aspect to Maven. 

Also, a typical dev workflow would be `make changes -> unit tests with testing protos -> integration tests with showcase/real protos`, this change will require the unit tests to dependent on showcase client, which feels backwards. Now the dev workflow would be `make changes -> build showcase client -> run both unit tests and integration tests`.